src/Text/Pandoc/Writers/OpenDocument.hs

```diff
@@ -46,13 +46,13 @@ import Data.Text (Text)
 import Text.Pandoc.BCP47 (Lang (..), parseBCP47)
 import Text.Pandoc.Class (PandocMonad, report, translateTerm,
                           setTranslations, toLang)
-import Text.Pandoc.Definition
+import Text.Pandoc.Definition as Definition
 import Text.Pandoc.Logging
 import Text.Pandoc.Options
 import Text.Pandoc.Pretty
 import Text.Pandoc.Shared (linesToPara)
 import Text.Pandoc.Templates (renderTemplate')
-import Text.Pandoc.Translations (Term(Figure))
+import Text.Pandoc.Translations as Term (Term(Figure, Table))
```

Comment:
Minor stylistic preference:  I think it would be nicer to do `import qualified ... as Term` and drop the need for `Definition.Table`.  But I wouldn't insist on this.